cmd/webhook/init/logging/log.go

```diff
@@ -22,14 +23,14 @@ func setLogFormat() {
 
 func setLogLevel() {
 	level := os.Getenv("LOG_LEVEL")
-	switch level {
-	case "debug":
+	switch {
+	case strings.EqualFold(level, "debug"):
```

Comment:
I like the intention, but do we really want to allow people to write `LOG_LEVEL=dEbUg`?

Reply:
![image](https://github.com/crutonjohn/external-dns-opnsense-webhook/assets/19677342/7c03120b-63eb-41cb-87e1-696c5571d99a)

Reply:
Yeah, and "Debug" and all other permutations. My view is I'd rather avoid punishing the user for not reading the docs and completely adhering to the rules for less critical things like this. It reduces the support burden too. Ideally just supporting `DEBUG` or `debug` would be the most reasonable but EqualFold already exists and does that part too.

 It does add a dependency which all else equal should be avoided but I think we can rely on "strings".